ovos_config/__main__.py

```
@@ -176,12 +176,13 @@
 
 @config.command()
 @click.option("--lang", "-l", required=True, help="the language code")
+@click.option("--platform", "-p", required=False, help="optimize for one of (rpi3, rpi4, rpi5, 32bit, linux, mac, windows)")
 @click.option("--hybrid", "-hy", is_flag=True, help="set default offline TTS and online STT plugins")
 @click.option("--online", "-on", is_flag=True, help="set default online TTS and STT plugins")
 @click.option("--offline", "-off", is_flag=True, help="set default offline TTS and STT plugins")
 @click.option("--male", "-m", is_flag=True, help="set default male voice for TTS")
 @click.option("--female", "-f", is_flag=True, help="set default female voice for TTS")
-def autoconfigure(lang, hybrid, online, offline, male, female):
+def autoconfigure(lang, platform, hybrid, online, offline, male, female):
```

Pylint (3.3.7)

[refactor] Too many arguments (7/5)

(R0913)